src/litdata/streaming/config.py

```diff
@@ -58,7 +60,7 @@ def __init__(
         self._config["data_spec"] = treespec_loads(self._config["data_spec"])
 
         self._item_loader.setup(self._config, self._chunks, serializers)
-        self._intervals = self._item_loader.generate_intervals()
+        self._intervals = self._item_loader.generate_intervals(self.subsample_interval)
```

Comment:
I don't think we need to pass it down to the item loader. I think it can just takes a list of chunks and intervals and remap them